js/notify/electron-notify.js

```diff
@@ -165,7 +165,12 @@ function updateConfig(customConfig) {
     if (customConfig.display) {
         displayId = customConfig.display;
     }
-    closeAll();
```

Comment:
Wasn't there a bug for which we had added this code? Please check this ticket -> https://perzoinc.atlassian.net/browse/ELECTRON-222

Reply:
That javascript error was taken care of in #326 and the change related to that fix is here -> https://github.com/KiranNiranjan/SymphonyElectron/blob/24d6176c772fd035c3796d9f5f7e4291ee62d2a8/js/notify/electron-notify.js#L715-L728